README.md

```diff
@@ -1,16 +1,33 @@
+With the addition of [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) to the browser, video player developers can create custom elements that mimic the video tag API, with goals of creating stand-in compatibility with the video tag and compatibility across players. The video tag API is however lacking some important functions to support modern player UIs, including playback quality/resolution selection and awareness of ads. This repo is intended to capture requests and proposals for those functions.
 
-## Submitting a proposal
-Use `template.md` to frame your proposal and make a pull-request to add your proposal document to the `proposals` directory.
+## Goals for Proposals
+we want these proposals to be something that we can eventually propose to the Media WG or WhatWG as additional features to the video element and anything related. In the meantime can be used as a spec to keep things interoperable between implementers.
+
+## Before submitting a proposal
+If you have a new idea, it might be worth creating an issue to discuss it a bit first before submitting a proposal to make sure that it's something could be a good fit and won't be less likely to be rejected later on.
 
-Inspired by [rust-lang/rfcs](https://github.com/rust-lang/rfcs) via [video-dev/hlsjs-rfcs](https://github.com/video-dev/hlsjs-rfcs).
+## Submitting a proposal
+- Fork the repo
+- Copy `0000-template.md` to `proposals/0000-your-feature.md`. Don't change the number yet as that will be updated when the PR is created.
+- Fill out your proposal with as much detail as possible.
+- Submit a pull request. Once the PR is created, the file `0000-` prefix and the header of the file should be updated to point at the new PR number.
+- Folks should now review the proposal and suggest changes. While in the PR, it's expected that many changes may be made. Ideally, please don't rebase commits in the branch as it'll make reviewing only new changes easier.
+- At some point, a team member should issue a Call for Consensus (CfC). This CfC will include whether this proposal should be accepted, rejected, or postponed.
+- The CfC will last a minimum of 2 weeks to make sure that everyone gets a chance to review
+- If the CfC generated a lot of new comments, the proposal will likely be rejected or go back to development mode again.
 
-## Submitting a request
-Use the github issues to request and disucss a desired feature/function at a high level.
+## Implementing a proposal
+Once a proposal has been accepted, it can start to be implemented. If during implementation, major issues are found, the existing proposal shouldn't be updated but a new proposal should be issued. The old proposal should be updated to link to the new proposal.
```

Comment:
suggestion(non-blocking): Should we instead formalize something like a "draft proposal" for an accepted proposal that has yet to be implemented? With our current directory structure and top level readme, it may be difficult to identify superseding proposals and the set of proposals that are
1. accepted
2. implemented
3. not superseded